### PR TITLE
#166816057 Setup react router

### PR DESCRIPTION
**Pull request by tunedev**
#### What does this PR do?
Set up react-router

 

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#166816057](https://www.pivotaltracker.com/story/show/166816057)

#### Screenshots (if appropriate)
react-router

#### Questions:
NA

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+import LoginPage from './views/LoginPage/LoginPage';
 
 class App extends Component {
   render() {
     return (
-      <div>
-        <h1>Team Persephone - Author's Haven</h1>
-      </div>
+      <Router>
+        <ul>
```
Comment (fxola) [inline src/App.js:11]: Great Job Tunde, I noticed you added some list elements on the app component, I'm a bit unclear about why you added them. Can you enlighten me on why you chose to add them?

Comment (tunedev) [reply src/App.js:11]: Thanks, @fxola, regarding the list elements, I added them to demonstrate the use case of the react-router. they are acting as the link to the demo pages I created. hope this is clear enough?

Comment (fxola) [reply src/App.js:11]: Oh Cool 👍🏼

**Review by fxola — APPROVED** [review]

**Comment by halimahO** [comment]
LGTM....Good job Tunde

**Pull request merged**